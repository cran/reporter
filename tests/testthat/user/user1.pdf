%PDF-1.7
%‚„œ”
1 0 obj<</Type /Catalog /Pages 4 0 R>>
endobj
2 0 obj<</Type /Font /Subtype /Type1 /BaseFont /Courier /Encoding /WinAnsiEncoding>>
endobj
3 0 obj<</Type /Font /Subtype /Type1 /BaseFont /Courier-Bold /Encoding /WinAnsiEncoding>>
endobj
4 0 obj<</Type /Pages /Kids [5 0 R] /Count 1 /MediaBox [0 0 792 612]>>
endobj
5 0 obj<</Type /Page /Parent 4 0 R /Contents 6 0 R /Resources <</Font <</F1 2 0 R /F2 3 0 R>> /ProcSet [/PDF /Text]>>>>
endobj
6 0 obj<</Length 5387>>
stream
BT /F1 10 Tf 87 Tz 67 535 Td <2020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020205461626c652031342e312f34>Tj ET
BT /F1 10 Tf 87 Tz 67 523.1 Td <202020202020202020202020202020202020202020202020202020202020202020202020202020202044656d6f677261706869637320616e6420426173656c696e6520746f20436861726163746572697374696373>Tj ET
BT /F1 10 Tf 87 Tz 67 511.2 Td <2020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020205370656369667920506f70756c6174696f6e>Tj ET
BT /F1 10 Tf 87 Tz 67 499.3 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 487.4 Td <202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020506c616365626f20202020202020202020447275672031306d67202020202020202020447275672032306d672020202020202020436f6d70657469746f72>Tj ET
BT /F1 10 Tf 87 Tz 67 475.5 Td <202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020284e3d333629202020202020202020202020284e3d333829202020202020202020202020284e3d333829202020202020202020202020284e3d333829>Tj ET
BT /F1 10 Tf 87 Tz 67 463.6 Td <2020202020202020202020202d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d2d>Tj ET
BT /F1 10 Tf 87 Tz 67 451.7 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 439.8 Td <20202020202020202020202041676520204e20202020202020202020202020202020202020202020202020202020202020203336202020202020202020202020202020203338202020202020202020202020202020203338202020202020202020202020202020203338>Tj ET
BT /F1 10 Tf 87 Tz 67 427.9 Td <20202020202020202020202020202020204d65616e2020202020202020202020202020202020202020202020202035312e37202831352e30292020202020202034362e39202831342e37292020202020202034392e31202831342e37292020202020202035302e39202831342e3629>Tj ET
BT /F1 10 Tf 87 Tz 67 416 Td <20202020202020202020202020202020204d656469616e202020202020202020202020202020202020202020202020202035322e35202020202020202020202020202034362e30202020202020202020202020202035302e35202020202020202020202020202035332e30>Tj ET
BT /F1 10 Tf 87 Tz 67 404.1 Td <20202020202020202020202020202020205131202d2051332020202020202020202020202020202020202020202034302e38202d2036302e352020202020202033352e30202d2035362e382020202020202034342e32202d2036312e302020202020202033392e32202d2036342e30>Tj ET
BT /F1 10 Tf 87 Tz 67 392.2 Td <202020202020202020202020202020202052616e676520202020202020202020202020202020202020202020202020203234202d20373520202020202020202020203232202d20373320202020202020202020203139202d20373120202020202020202020203231202d203735>Tj ET
BT /F1 10 Tf 87 Tz 67 380.3 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 368.4 Td <20202020202020202020202053657820204d616c6520202020202020202020202020202020202020202020202020323420282036362e37252920202020202020313820282034372e34252920202020202020323120282035352e33252920202020202020323620282036382e342529>Tj ET
BT /F1 10 Tf 87 Tz 67 356.5 Td <202020202020202020202020202020202046656d616c652020202020202020202020202020202020202020202020313220282033332e33252920202020202020323020282035322e36252920202020202020313720282034342e37252920202020202020313220282033312e362529>Tj ET
BT /F1 10 Tf 87 Tz 67 344.6 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 332.7 Td <20202020202020202020202052616365205768697465202020202020202020202020202020202020202020202020333120282038362e31252920202020202020333220282038342e32252920202020202020333220282038342e32252920202020202020333520282039322e312529>Tj ET
BT /F1 10 Tf 87 Tz 67 320.8 Td <2020202020202020202020202020202020426c61636b206f72204166726963616e20416d65726963616e202020203320282020382e33252920202020202020203320282020372e39252920202020202020203620282031352e38252920202020202020203120282020322e362529>Tj ET
BT /F1 10 Tf 87 Tz 67 308.9 Td <2020202020202020202020202020202020417369616e206f7220506163696669632049736c616e646572202020203120282020322e38252920202020202020203220282020352e33252920202020202020203020282020302e30252920202020202020203120282020322e362529>Tj ET
BT /F1 10 Tf 87 Tz 67 297 Td <2020202020202020202020202020202020556e6b6e6f776e202020202020202020202020202020202020202020203120282020322e38252920202020202020203120282020322e36252920202020202020203020282020302e30252920202020202020203120282020322e362529>Tj ET
BT /F1 10 Tf 87 Tz 67 285.1 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 273.2 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 261.3 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 249.4 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 237.5 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 225.6 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 213.7 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 201.8 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 189.9 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 178 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 166.1 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 154.2 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 142.3 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 130.4 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 118.5 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 106.6 Td <5370656369616c2073796d626f6c7320b5203f20a320746865726520746f206d657373207468696e67732075703a>Tj ET
BT /F1 10 Tf 87 Tz 67 94.7 Td <>Tj ET
BT /F1 10 Tf 87 Tz 67 82.8 Td <54696d6520202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020506167652031206f662031>Tj ET
endstream
endobj
7 0 obj<</Producer (reporter v1.2.7) /Author (dbosa) /Title () /Subject () /Creator (R version 4.1.2 (2021-11-01)) /Keywords ()
 /CreationDate (D:20211209213149-05'00') /ModDate (D:20211209213149-05'00')>>
endobj
xref
0 8
0000000000 65535 f 
0000000015 00000 n 
0000000061 00000 n 
0000000153 00000 n 
0000000250 00000 n 
0000000328 00000 n 
0000000455 00000 n 
0000005890 00000 n 
trailer <</Size 8 /Root 1 0 R /Info 7 0 R>>
startxref
6103
%%EOF