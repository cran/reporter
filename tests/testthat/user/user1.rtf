{\rtf1\ansi\deff0 {\fonttbl {\f0 Courier;}}
\landscape\horzdoc
\paperw15840\paperh12240
\margl1440\margr1296\margt720\margb360
\fs20
                                                Table 14.1/4\line
                                Demographics and Baseline to Characteristics\line
                                             Specify Population\line
\line
                                             Placebo     Drug 10mg    Drug 20mg    Competitor\line
                                              (N=36)       (N=38)       (N=38)       (N=38)\line
            \u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\u175\line
\line
            Age  N                              36           38           38           38\line
                 Mean                      51.7 (15.0)  46.9 (14.7)  49.1 (14.7)  50.9 (14.6)\line
                 Median                        52.5         46.0         50.5         53.0\line
                 Q1 - Q3                   40.8 - 60.5  35.0 - 56.8  44.2 - 61.0  39.2 - 64.0\line
                 Range                       24 - 75      22 - 73      19 - 71      21 - 75\line
\line
            Sex  Male                      24 ( 66.7%)  18 ( 47.4%)  21 ( 55.3%)  26 ( 68.4%)\line
                 Female                    12 ( 33.3%)  20 ( 52.6%)  17 ( 44.7%)  12 ( 31.6%)\line
\line
            Race White                     31 ( 86.1%)  32 ( 84.2%)  32 ( 84.2%)  35 ( 92.1%)\line
                 Black or African American  3 (  8.3%)   3 (  7.9%)   6 ( 15.8%)   1 (  2.6%)\line
                 Asian or Pacific Islander  1 (  2.8%)   2 (  5.3%)   0 (  0.0%)   1 (  2.6%)\line
                 Unknown                    1 (  2.8%)   1 (  2.6%)   0 (  0.0%)   1 (  2.6%)\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
\line
Special symbols \u181   \u8486   \u163   there to mess things up:\line
\line
Time                                                                                             Page 1 of 1\line
}
