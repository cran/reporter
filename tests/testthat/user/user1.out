                                                Table 14.1/4
                                Demographics and Baseline to Characteristics
                                             Specify Population

                                             Placebo     Drug 10mg    Drug 20mg    Competitor
                                              (N=36)       (N=38)       (N=38)       (N=38)
            ¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯

            Age  N                              36           38           38           38
                 Mean                      51.7 (15.0)  46.9 (14.7)  49.1 (14.7)  50.9 (14.6)
                 Median                        52.5         46.0         50.5         53.0
                 Q1 - Q3                   40.8 - 60.5  35.0 - 56.8  44.2 - 61.0  39.2 - 64.0
                 Range                       24 - 75      22 - 73      19 - 71      21 - 75

            Sex  Male                      24 ( 66.7%)  18 ( 47.4%)  21 ( 55.3%)  26 ( 68.4%)
                 Female                    12 ( 33.3%)  20 ( 52.6%)  17 ( 44.7%)  12 ( 31.6%)

            Race White                     31 ( 86.1%)  32 ( 84.2%)  32 ( 84.2%)  35 ( 92.1%)
                 Black or African American  3 (  8.3%)   3 (  7.9%)   6 ( 15.8%)   1 (  2.6%)
                 Asian or Pacific Islander  1 (  2.8%)   2 (  5.3%)   0 (  0.0%)   1 (  2.6%)
                 Unknown                    1 (  2.8%)   1 (  2.6%)   0 (  0.0%)   1 (  2.6%)





















Special symbols µ Ω £ there to mess things up:

Time                                                                                             Page 1 of 1
